tests/actions/test_core_actions.py

```diff
@@ -462,8 +469,6 @@ def test_MicrosoftTeamsNotificationAction_bad_request(
         validation_result_suite=validation_result_suite,
     ) == {"microsoft_teams_notification_result": None}
 
-    assert "Request to Microsoft Teams webhook returned error 400" in caplog.text
```

Comment:
Should we still make this assert? It doesn't look like the rest of this method changed.
If we do remove it, you can remove the `caplog` fixture from the arguments to this method and setting the loglevel.